pages/version-policy/index.md

```diff
@@ -430,6 +430,18 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">◯</p></td>
         <td><p style="text-align: center;">◯</p></td>
+    </tr>
+     <tr>
+        <td>Kubernetes Cluster 2.1.x-1.12.y</td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">◯</p></td>
+        <td><p style="text-align: center;">◯</p></td>
+    </tr>
+    <tr>
+        <td>Kubernetes Cluster 2.2.x-1.13.y (Recommended)</td>
```

Comment:
I do see newer versions listed here https://docs.mesosphere.com/services/kubernetes/. Should this PR be updated with newer versions?
![Screen Shot 2019-04-29 at 10 28 02 AM](https://user-images.githubusercontent.com/36425832/56914655-80693680-6a69-11e9-90e0-61be02a55485.png)

Reply:
That's up to @chrisgaun (but he's on Paternity Leave now)